HiP-DataStore.Model/Entity/Tag.cs

```diff
@@ -9,6 +9,8 @@ public class Tag : ContentBase
 
         public string Title { get; set; }
 
+        public bool IsUsed { get; set; }
```

Comment:
Property should be removed. Use `tag.Referencees.Count > 0` to check if a tag is in use.